apps/wallets/smart-wallet/react/README.md

```diff
@@ -52,3 +57,51 @@ pnpm run dev
 ```
 
 This command will launch the application on `http://localhost:3000` by default, allowing you to view and interact with it in your web browser.
+
+## Playwright setup
+
+Authenticating with Google auth requires adding this file to the project.
+
+add `playwright/.auth/user-session.json` to the root of this app.
```

Comment:
Can you generate this programmatically?

Reply:
turns out it already does... :sweat_smile: remooving this in readme